dev/release_notes.readme.md

```diff
@@ -0,0 +1,102 @@
+# How to produce release notes for GAP essentially automatically
```

Comment:
Maybe we can make a short summary at the beginning of how to deal with PR in the future if somebody do not want to know the reason or explanations for the changes.

Reply:
@danielrademacher Sorry, I do not catch the point. What do you suggest to add where in this file?

Reply:
The text here is excellently written, summarizes all the important points from the GAP Days and provides a great overview for the future process. I just thought that for someone who would like to join the GAP community from scratch or maybe just want to start a PR, a short summary of how the labels should be dealt with would be helpful. When I think back to my first GAP Days, I was partially overwhelmed by all the processes and details. It was just an idea to possibly simplify the start.

Reply:
@ThomasBreuer updated the `.github/pull_request_template.md` file and added such a brief summary, I think that does it? Other than that, I think people who submit PRs need not worry much about labels, it is people who merge PRs who need to worry about it (and perhaps request help from the PR authors, with instructions).

That said, I guess we still do need a text somewhere which summarizes which labels to use how, targeted at the group of people who want to / are supposed to review/triage/manage/merge PRs. But doing so goes beyond this PR, so I don't expect Thomas to add it here. But @danielrademacher feel free to open an issue requesting that and/or a PR starting such a file (it can be incomplete, and me and others can add more data to it in review comments for the PR, or after it got merged).

Related: issue #3381

Reply:
Sorry, I didn't see the file ```.github/pull_request_template.md```. The changes are great, that's exactly what I was thinking of. Thank you!